microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -60,10 +61,49 @@ The Microsoft Edge team will independently decide on MV3 migration timelines for
 | TBD | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge, by using enterprise policies. |
 | TBD | No change. | Manifest V2 extensions will no longer function in Microsoft Edge, even with the use of enterprise policies. |
 
-Manifest V2 extensions will continue to be supported through Enterprise policies at least until the date in the Chromium Manifest V2 support timeline.
+<!-- todo: existing MV2 extensions already published on the Microsoft Edge Add-ons website can be updated until the end of FY-24. [what Month/Day?]  This timeline is subject to change. -->
+
+<!-- todo: at TBDs, add "see below"?  or, replace right-hand cell content by the main point that's below? -->
+
+
+<!-- ------------------------------ -->
+#### Timeline for ending support for Manifest V2-based Microsoft Edge extensions for enterprises
+
+<!-- todo: orig draft wording, for reference:
+about MV2 TimeLine for Enterprise:
+ 
+We see the move to MV3 as a change in the positive direction to improve security, privacy and performance of extensions.
+Our current strategy is to honour any new mv3 extensions from our partner ecosystem and any updates to existing mv2 extensions.
+At the same time, we will continue updates for MV2 extensions for an extended period of time and will communicate once we plan deprecation of existing mv2 extensions.
+For enterprises, Manifest V2 extensions will continue to be supported through Enterprise policies at least until the date in the Chromium Manifest V2 support timeline.
+-->
+
+The move to Manifest V3 improves the security, privacy, and performance of extensions.<!-- todo: delete sentence about MV3, b/c off-topic for this "MV2" section -->
+Our current strategy for Microsoft Edge is to support new MV3-based extensions and support updates to existing MV2-based extensions from our partner ecosystem.<!-- todo: delete sentence b/c the mv3 point is off-topic for this "MV2" section, & the mv2 point is redundant w/ next, clearer sentence --><!-- todo: delete "from our partner ecosystem" b/c it's undefined & redundant or superfluous; not needed: can there exist any existing MV2-based extensions that are not from our partner ecosystem? -->
+Microsoft Edge Extensions will continue to support updates for MV2-based extensions for an extended period of time, and we will communicate with extension developers when we plan the deprecation of existing MV2-based extensions.<!-- todo: move this sentence to be first -->
+For enterprises, Manifest V2 extensions will continue to be supported through Enterprise policies at least until the date in the Chromium Manifest V2 support timeline.<!-- todo: revise phrase "at least ... timeline" -->
```

Comment:
revise the phrase "at least until the date in the Chromium Manifest V2 support timeline", b/c that's the problem that this PR is intended to fix & clarify.
@sivMSFT 

that's the sentence that appears in live docs
https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/manifest-v3#manifest-timeline-for-microsoft-edge-and-partner-center